vsintegration/tests/UnitTests/VisualFSharp.UnitTests.fsproj

```diff
@@ -56,56 +56,9 @@
     <Compile Include="LegacyProjectSystem\Tests.ProjectSystem.References.fs" />
     <Compile Include="LegacyProjectSystem\Tests.ProjectSystem.RoundTrip.fs" />
     <Compile Include="LegacyProjectSystem\Tests.ProjectSystem.UpToDate.fs" />
-    <Compile Include="..\..\..\tests\service\FsUnit.fs">
-      <Link>CompilerService\FsUnit.fs</Link>
-    </Compile>
-    <Compile Include="..\..\..\tests\service\Common.fs">
-      <Link>CompilerService\Common.fs</Link>
-    </Compile>
-    <Compile Include="..\..\..\tests\service\Symbols.fs">
-      <Link>CompilerService\Symbols.fs</Link>
-    </Compile>
-    <Compile Include="..\..\..\tests\service\EditorTests.fs">
-      <Link>CompilerService\EditorTests.fs</Link>
-    </Compile>
-    <Compile Include="..\..\..\tests\service\FileSystemTests.fs">
-      <Link>CompilerService\FileSystemTests.fs</Link>
-    </Compile>
-    <Compile Include="..\..\..\tests\service\ProjectAnalysisTests.fs">
-      <Link>CompilerService\ProjectAnalysisTests.fs</Link>
-    </Compile>
-<!-- TODO:  Fix this test
-    <Compile Include="..\..\..\tests\service\MultiProjectAnalysisTests.fs">
-      <Link>CompilerService\MultiProjectAnalysisTests.fs</Link>
-    </Compile>
--->
-    <Compile Include="..\..\..\tests\service\PerfTests.fs">
-      <Link>CompilerService\PerfTests.fs</Link>
-    </Compile>
-    <Compile Include="..\..\..\tests\service\InteractiveCheckerTests.fs">
-      <Link>CompilerService\InteractiveCheckerTests.fs</Link>
-    </Compile>
-    <Compile Include="..\..\..\tests\service\ExprTests.fs">
-      <Link>CompilerService\ExprTests.fs</Link>
-    </Compile>
-    <Compile Include="..\..\..\tests\service\CSharpProjectAnalysis.fs">
-      <Link>CompilerService\CSharpProjectAnalysis.fs</Link>
-    </Compile>
-    <Compile Include="..\..\..\tests\service\StructureTests.fs">
-      <Link>CompilerService\StructureTests.fs</Link>
-    </Compile>
-    <Compile Include="..\..\..\tests\service\AssemblyContentProviderTests.fs">
-      <Link>CompilerService\AssemblyContentProviderTests.fs</Link>
-    </Compile>
-    <Compile Include="..\..\..\tests\service\ServiceUntypedParseTests.fs">
-      <Link>CompilerService\ServiceUntypedParseTests.fs</Link>
-    </Compile>
```

Comment:
This disables running these tests from VS locally. Why remove them?

Reply:
They got moved to FSharp.COmpiler.UnitTests